server.js

```diff
@@ -1136,6 +1136,12 @@ if (config.startServer) {
                 callback(null);
             });
         },
+        function(callback) {
```

Comment:
I'd appreciate a few lines of comments here talking about why we do this and the importance of the sequence between migrations, random schema, and sproc creation.

Reply:
Excellent point. I added both a comment here in the code and also docs in `docs/dev-guide.md`.